doc/CLI-user-manual/README.md

```diff
@@ -512,35 +527,37 @@ The credential operations below meet the specifications defined in [W3C VC Data
-Carol will grant Bob access only if Bob can prove that he owns a particular credential issued by Alice.
+The basic workflow involves three actors: Alice (the Issuer), Bob (the Holder), and Carol (the Verifier). In this scenario, Bob wishes to gain access to some resource controlled by Carol. Carol will grant Bob access only if Bob can prove that he owns a particular credential issued by Alice.
 
 #### Steps to Creating a VC:
 
-1. Issuer creates Credential. The Credential is like a type or a class, it describes the Credential, specifies a schema for the Credential. Each Verified Credential is an instance of a Credential.
-1. Issuer binds a Credential to a Holder
-1. Issuer attest a Credentials, creating a Verifiable Credential,  by signing and encrypting a Bound Credential.
-1. Holder accepts the Verified Credential (adding it to their wallet for future use).
-1. Verifier creates a Challenge. A Challenge is a list of Credentials and trusted Issuers.
-1. Holder creates a Responses to Challenge. A Response contains a list of Verified Presentations that correspond to the Credentials and trusted Issuers listed in the Challenge. If the User's Wallet contains the right Verified Credentials, then it will be possible to create a valid Response to the Challenge. The Response is encrypted for the Verifier.
-1. Verifier verifies the Response. The Response is validated by checking that it contains a Verified Presentation for every Credential in the Challenge, and that the VP is issued by a trusted Issuer, and that the VC corresponding the VP has not been revoked. If everything checks out the Response is accepted and the Holder is granted authorization.
+1. The Issuer (Alice) creates a Credential. The Credential is like a type or a class; it describes the Credential, and specifies a schema. Each Verified Credential is an instance of a Credential.
+1. The Issuer binds a Credential to a Holder
+1. The Issuer attest a Credential, creating a Verifiable Credential (VC),  by signing and encrypting a Bound Credential.
+1. The Holder (Bob) accepts the VC (adding it to their wallet for future use).
+1. The Verifier (Carol) creates a Challenge. A Challenge is a list of Credentials and trusted Issuers.
+1. The Holder creates a Response to the Challenge. A Response contains a list of Verified Presentations that correspond to the Credentials and trusted Issuers listed in the Challenge. If the User's wallet contains the right Verified Credentials, then it will be possible to create a valid Response to the Challenge. The Response is encrypted for the Verifier.
+1. The Verifier verifies the Response. The Response is validated by checking that it contains a Verified Presentation for every Credential in the Challenge, and that the VP is issued by a trusted Issuer, and that the VC corresponding the VP has not been revoked. If everything checks out the Response is accepted and the Holder is granted authorization.
```

Comment:
Yes. In fact, I'd like to use it more to avoid redundant word use on L554:

```suggestion
1. The Verifier validates the Response. The Response is verified by checking that it contains a Verified Presentation for every Credential in the Challenge,  that the VP is issued by a trusted Issuer, and that the VC corresponding the VP has not been revoked. If everything checks out the Response is accepted and the Holder is granted authorization.
```